context/app/static/js/components/Footer/Footer.jsx

```diff
@@ -48,7 +48,9 @@ function Footer() {
               >
                 Data Use Agreement
               </LightBlueLink>
-              <LightBlueLink variant="body2">Citing HuBMAP</LightBlueLink>
+              <LightBlueLink href="/docs/about" variant="body2">
```

Comment:
```suggestion
              <LightBlueLink href="/docs/about#citation" variant="body2">
```
Anchors don't work yet, but I'd like to get them in.